core/services/ocr2/plugins/mercury/integration_plugin_test.go

```diff
@@ -6,17 +6,19 @@ import (
 	"testing"
 
 	"github.com/smartcontractkit/chainlink/v2/core/config/env"
+	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 )
 
 func TestIntegration_MercuryV1_Plugin(t *testing.T) {
+	testutils.SkipFlakey(t, "https://smartcontract-it.atlassian.net/browse/MERC-5697")
```

Comment:
The link in the ticket is for `TestIntegration_MercuryV1`

Reply:
Could the owner of the ticket create subtasks? I assume that all flakes will be handled in the same sprint/context.

Reply:
I mean that you have marked the wrong test for skipping. Do you have a link to an instance of `TestIntegration_MercuryV1_Plugin` failling? The existing link is for `TestIntegration_MercuryV1`

Reply:
You are absolutely right 🙈 updated